apps/penxle.com/src/routes/(default)/[space]/Post.svelte

```diff
@@ -829,7 +837,14 @@
   </div>
 </article>
 
-<LoginRequireModal bind:open={loginRequireOpen} />
+<LoginRequireModal bind:open={loginRequireOpen}>
+  <svelte:fragment slot="subtitle">
+    {$query.post.ageRating === 'R15' ? '15세' : '성인'} 포스트 열람을 위해 로그인이 필요해요
+  </svelte:fragment>
+  <svelte:fragment slot="action-secondary">
+    <Button class="w-full" color="secondary" size="xl" on:click={() => history.back()}>뒤로 가기</Button>
```

Comment:
P4: 이 경우 실행할 함수 자체를 값으로 주입하면 어떨까요?

```suggestion
    <Button class="w-full" color="secondary" size="xl" on:click={history.back}>뒤로 가기</Button>
```

Reply:
`history.back()` 함수는 인수가 없어서 안전하긴 한데... 그래도 일관성을 위해서 (인수가 있는 함수 자체를 값으로 넣으면 정의되지 않은 동작을 할 가능성이 있으니까) 지금처럼 익명 함수로 감싸는 편이 좀 더 나을 거 같아요